website/docs/docs/dbt-cloud/access-control/self-service-permissions.md

```diff
@@ -18,3 +16,20 @@ dbt Cloud supports two different permission sets to manage permissions for self-
 | Manage billing | ❌ | ✅ |
 | Manage team permissions | ❌ | ✅ |
 | Invite Owners to the account | ❌ | ✅ |
+
+## Member and Owner Groups in dbt Cloud Enterprise
+
+By default, new users are added to the Member and Owner groups when they onboard to dbt Cloud. This is necessary for account onboarding but can create confusion when initially setting up SSO and RBAC for dbt Cloud Enterprise accounts as described in the [Enterprise Permissions](https://docs.getdbt.com/docs/dbt-cloud/access-control/enterprise-permissions/) guide. Users may be added with more permissions than intended without adjusting this setting.
```

Comment:
Could this link to the actual object (don't know what that's called but I think it should be `dbt-cloud/access-control/enterprise-permissions`)

Reply:
>Users may be added with more permissions than intended without adjusting this setting.

The key issue with having these groups is that these groups override all the other permission sets because they're both account level permissions sets. I would emphasize that this needs to be removed because of the potential creation of conflicting permission sets.

Reply:
Unclear which object you're referring to and how that's different from the link. Do you mean the diagram within the linked page?

Reply:
@amychen1776 I'm not following what you mean by the actual object and how that's different from providing a link to the relevant page. Can you please elaborate?

Reply:
Yeah, let me see if I can try to explain better with my limited knowledge of markdown. The better practice is to use a relative link rather than one that links directly to the production page. This way, when you host the website locally, your links still work rather than redirecting you to the production page. You want to link to the file in the repository.

Reply:
Ah ok I see what you mean now! Thanks for describing that more. Changed this as well.